src/server/index.js

```diff
@@ -116,7 +116,10 @@ async function createServer() {
     nunjucksConfig,
     sanitise,
     router,
-    auditing
+    {
+      plugin: auditor,
+      options: { audit: config.get('audit') }
```

Comment:
@flurdy audit is being sent here via options

Reply:
Cool